php/EE/Runner.php

```diff
@@ -173,7 +173,7 @@ public function get_packages_dir_path() {
 		if ( getenv( 'EE_PACKAGES_DIR' ) ) {
 			$packages_dir = Utils\trailingslashit( getenv( 'EE_PACKAGES_DIR' ) );
 		} else {
-			$packages_dir = Utils\get_home_dir() . '/.ee4/packages/';
+			$packages_dir = Utils\get_home_dir() . '/.ee/packages/';
```

Comment:
Shouldn't the packages also go in the common global easyengine config directory?

Reply:
home dir packages folder will only make sense if we want to support per user package.

but default should be `/opt/easyengine/packages/`